Controller/ForgotResetController.php

```diff
@@ -93,19 +95,19 @@ public function forgotPasswordAction(Request $request)
 
                     $request->getSession()->getFlashBag()->add(
                         "success",
```

Comment:
In addition to using the same message, we should use the same type of flash bag to avoid hinting that an email address exists. Consider using the "success" type for all cases. (This probably means code could be further refactored to avoid duplicate code.)